spec.md

```diff
@@ -215,6 +215,30 @@ help intermediate gateways determine how to route the events.
     * urn:event:from:myapi/resourse/123
     * mailto:cncf-wg-serverless@lists.cncf.io
 
+### subject
+* Type: `String`
+* Description: This describes the subject of the event in the context of the
+  event producer (identified by `source`). In publish-subscribe scenarios,
+  a subscriber will typically subscribe to events emitted by a `source`,
+  but the `source` identifier alone might not be sufficient as a qualifier for
+  any specific event if the `source` context has internal sub-structure.
+* Example: A subscriber might register interest for when new blobs are created
+  inside a blob-storage container. In this case, the event `source` identifies
+  the subscription scope (storage container), the `type` identifies the "blob
+  created" event, and the `id` uniquely identifies the event instance to
+  distinguish separate occurrences of a same-named blog having been created;
+  the name of the newly created blob is carried in `subject`.
+
+  Identifying the subject of the event in context metadata (opposed to only in
+  the `data` payload) is particularly helpful in generic subscription filtering
+  scenarios where middleware is unable to interpret the `data` content. In the
+  above example, the subscriber might only be interested in blobs with names
+  ending with '.jpg' or '.jpeg' and the subject attribute allows for constructing
+  a simple and efficient string-suffix filter for that subset of events.
+* Constraints:
+  * OPTIONAL
+  * MUST be a non-empty string
```

Comment:
**If Present,** MUST be a non-empty string

Reply:
ok

Reply:
no pushy?